knowledge-base/actions/eberlitz/lcov-reporter-action/action-security.yml

```diff
@@ -0,0 +1,12 @@
+name: "Code Coverage Report"
+github-token:
+  action-input:
+    input: github_token
+    is-default: true
+  permissions:
```

Comment:
`is-default` will be false, view [here](https://github.com/eberlitz/lcov-reporter-action/blob/026f884589a418aafc917b03e3c6d096c6ba808a/action.yml#L8)

Permission for this action will be:
`content: write`, `to create commit comment with code coverage`, view [here](https://github.com/eberlitz/lcov-reporter-action/blob/026f884589a418aafc917b03e3c6d096c6ba808a/src/index.js#L75), [here](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#commit-comments-1)
`issues: write`, `to comment code coverage on pr`, view [here](https://github.com/eberlitz/lcov-reporter-action/blob/026f884589a418aafc917b03e3c6d096c6ba808a/src/index.js#L68)

please update them